### PR TITLE
Botany adjustments / re-additions / fixes / whatever

### LINKED ISSUE #6407: Deep-frying an ear of corn won't make corn chips
**Problem Description**:
Putting an ear of corn into the deep fryer produces deep-fryed ear of corn instead of corn chips like the code and wiki say it is supposed to.

**What did you expect to happen**: 
Corn put in the deep fryer should become corn chips.

**What happened instead**:
Deep fryed corn was produced instead of corn chips.

**Why is this bad/What are the consequences:**
The code says that the corn should become chips, though the botany update might have changed the corn id or something along those lines.

**Steps to reproduce the problem**:
Get an ear of corn and put it into the deep fryer, then wait for it to come out.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: I found this in the code for the deep fryer:
86 /datum/deepfryer_special/corn_chips 
87 	input = "corn" 
88 	output = /obj/item/weapon/reagent_containers/food/snacks/cornchips

### PR DESCRIPTION
**Pull request by FalseIncarnate**
Fixes #6407 
- Corn and bananas will now properly be deepfried into corn chips and fried bananas respectively.
  - Variants of these plants (like mimanas) also will work as a substitute due to how subtyping works. This is fine / intentional.

Re-adds Tray Lids
image
- Hydroponics trays once again have tray lids which can be toggled via a verb or alt-click on the tray. These lids only protect the plant from accidental harvesting and bees. Especially bees.
  - Lid lights are not re-added, because they no longer are necessary.

Re-slows Plant Aging
- Plant aging has been readjusted to better reflect the longer average shift of Paradise in comparison to TG station.
  - Instead of a plant's age increasing every cycle, it once again increases every 3rd cycle (meaning a plant takes 3x as many cycles to reach maturity than it does currently).
  - This was how aging worked before the rework, so the pacing should feel familiar to veteran botanists and be less hectic for newcomers.

:cl:
bugfix: Corn and bananas can be properly deepfried into their special results again.
rscadd: Hydroponics trays once again are equipped with Bee-proof lids to keep unwanted pollination to a minimum. This only works if you actually toggle the lid closed though!
tweak: Plants age more slowly, so your space farming should no longer feel like it outpaces a meth'd up blue hedgehog in sneakers.
/:cl:

### CODE REVIEW

**Comment by Spacemanspark** [comment]
:+1:

**Comment by uraniummeltdown** [comment]
Staring at your screen waiting 3 times as long for plants to grow is not very fun, if you must make aging slower make it age every 2 cycles

**Comment by FalseIncarnate** [comment]
Adjusting the cycles per age is a simple tweak, but I went with 3 because it is both how the rate was previously and a rough estimation of how many times longer our rounds last compared to TG so it feels proportional.

That said, if enough people would prefer a different rate, I am willing to adjust it.

**Comment by Fox-McCloud** [comment]
Rounds on TG last 45-60 minutes; their more RP oriented server usually is 1-1.5 hours.

File: code/modules/hydroponics/hydroponics.dm [hunk 1]
```diff
@@ -993,6 +1009,20 @@
 	else
 		..()
 
+/obj/machinery/hydroponics/verb/toggle_lid_verb()
```
Comment (Fox-McCloud) [inline code/modules/hydroponics/hydroponics.dm]: Just kill this; verbs are disgusting; the alt-click will be enough.

**Comment by Fox-McCloud** [comment]
Merge conflicts.

I'm still not exactly keen on the speed adjustments for reasons stated above, especially with how the current pace gives something for people to do a lot more; it also gives player the very real option to focus on what the chef needs and wants for part of the shift and to engage in a lot of research during the latter part of the shift.

The botany lids thing is ok, though comes across as a bit pointless.

**Comment by FalseIncarnate** [comment]
Closing to re-upload because .dmi conflicts are a pain and I also want to split one of the commits into two more atomic ones anyways.

**Pull request closed without merging**